src/controllers/Auth.js

```diff
@@ -39,13 +44,13 @@ class Auth {
         role,
         companyId,
       });
+
       const response = new Response(
         true,
         201,
         'User signup successfully',
-        { user: { email, token }, }
+        { user: { email, token } }
       );
-
       return res.status(response.code).json(response);
     } catch (err) {
       const response = new Response(
```

Comment:
I think you missed something here. Or I missed something🤷‍♂

Reply:
You mean i should remove the empty line?, Nothing is missing.

Reply:
Never mind